src/main/scala/com/betting/domain/Event.scala

```diff
@@ -0,0 +1,34 @@
+package com.betting.domain
+
+import io.circe.syntax.EncoderOps
+import io.circe.{Decoder, Encoder, Json}
+import spray.json.DefaultJsonProtocol
+
+final case class Event(
+                        id: Long,
+                        name: String,
+                        inPlay: Boolean,
+                        startTime: String,
+                        endTime: String
+                      )
+
+trait EventProtocols extends DefaultJsonProtocol {
```

Comment:
using circe this marshalling an unmarshaling can be rewrite 
`implicit val codec: Codec[Event] = deriveCodec` in an companoin object no need for EventProtocols trait , and that's all no need for explicit json manipulation , this line will replace both marshaller and unmarshaller check circe documentation for reference [https://circe.github.io/circe/parsing.html](url)